src/controller/BallotController.ts

```diff
@@ -0,0 +1,15 @@
+import {Request} from "express";
+import {asyncHandler} from "../utils/AsyncHandler";
+import {BallotHandler} from "../handler/BallotHandler";
+
+const ballotHandler = new BallotHandler()
+
+export default {
+  list: asyncHandler(async (req: Request<{}, {}, {}>, res, next) => {
```

Comment:
Warum hast du hier im Controller Funktionsnamen? 

Fände hier ein get bzw. post besser. Der Controller ist ja wirklich dann schon im Prinzip die Verbindung nach draußen

Reply:
Wie soll man dann das klassische list + detail handhaben
Beispiel:
/api/blog -> Liste aller blogs
/api/blog/id -> Spezifischer Blog

Sind ja beides GET aber in meinen Augen der gleiche Controller

Reply:
Hätte gesagt sind zwei unterschiedliche Controller.
Vor allem dafür: Als User hast du ja vllt recht deinen eigenen wieder abzufragen, aber als Admin darfst du alle abfragen. Deswegen hätte ich jetzt gesagt sind das zwei unterschiedliche Controller weil sie auch unterschiedliche Berechtigungen haben können

Reply:
Der Ansatz der mir gerade dafür einfallen würde wäre es quasi per sub-route einen Controller zu bauen
-> api/blog -> BlogListController (GET, POST)
-> api/blog/id -> BlogDetailController (GET, PUT, PATCH)

die routes/index.ts würde dann wie folgt aussehen
![image](https://user-images.githubusercontent.com/58269084/152854314-d386863b-cf2c-48cf-83b6-60828e3077ee.png)

Dabei stört mich nur so Helper-Controller wie TokenStatus, da er aus dem klassischen REST-Model-Schema raus fällt und für immer ein einsamer GET bleibt 😢.

Reply:
Och finde den ersten Ansatz am besten zum Lesen - das andere verwirrt imo

Reply:
Denke aber dein neuer Ansatz wäre der eigentlich richtige...
Aber wir können es auch so lassen, sollte bei beidem eigentlich verstanden werden